addons/cds-connector-0.0.1/chart/cds-connector/values.yaml

```diff
@@ -0,0 +1,7 @@
+# Default values for github-connector.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+apiRegistrationJobName: api-registration-job
+oAuthProxyName: oauth-proxy-connector
+namespace: kyma-integration
```

Comment:
The namespace is used for some resources, not for all. Is that really intended behavior? Shouldn't it all be located in the same namespace, for example the kyma-oauth-proxy deployment has no namespace specified and will end up in a different one.